src/apps/integrations/router.py

```diff
@@ -30,3 +30,14 @@
         **AUTHENTICATION_ERROR_RESPONSES,
     },
 )(retrieve_integration)
+
+router.delete(
+    "/",
```

Comment:
We usually follow a different convention for urls instead of passing params
`/integrations/applet/{applet_id}/integrations/{integration_id}`

You can check other routers for review, like Answers, Applets, Schedule, etc

Reply:
I understand this. I think we can retrieve the integrations_id then in the retrieval endpoint. Once we have that then we can delete by integrations_id.

Reply:
I found there is no need to add the id in the domain object. Currently I look the integration by applet_id and type first and then I use the integrations id to delete it. Please review.